js/schematic.js

```diff
@@ -176,7 +181,7 @@
   function setupSignals(dataset) {
 
       signal.on("downloadVisibleData", function (){
-          var selectedColumn = grid.selectedColumn()
+          var selectedColumn = query.question;
```

Comment:
Aha! My bad for not using this earlier. I didn't "get it" that `question` is conceptually the same as `selectedColumn`.